### ISSUE TITLE
Package ng-cordova disappeared

### ISSUE THREAD

**Issue by mvaneijk** [TbLtzk/Centaurus#68]
ngCordova is still there, I guess it is the same, but should be replaced in bower than

**Comment by TbLtzk** [comment]
Yes I noticed, when I did the apk reduction! But failed to use ngCordova instead. If this blocks your work we could consider checking the old ng-cordova into `lib_static` instead of bower reference
